docs/developer/other/IDE_INTEGRATION.md

```diff
@@ -0,0 +1,31 @@
+# IDE Integration
+
```

Comment:
> How is it better to do this rather than just open the project via IDE without having first done these steps?

In olden days it was because IDE imports for gradle missed a lot (things like subprojects would be ignored).
Now there isn't much difference whether use using the plugin or use import.

> Does it document anything that Idea and/or Eclipse users would be unlikely to know?

Unlikely, this would more likely help a developer new to either of the IDEs.

----

Taking the conversation a different direction.
The main reason I even considered adding this was to note a piece of functionality which is not normally run as part of the `gradle install`, developers have to go out of their way to run `gradle idea` or `gradle eclipse` to leverage it.

Would a better approach be to remove both these plugins entirely?